jax/_src/random.py

```diff
@@ -2030,6 +2030,8 @@ def orthogonal(
 ) -> Array:
   """Sample uniformly from the orthogonal group O(n).
 
+  Uses the technique described in https://arxiv.org/abs/math-ph/0609050.
```

Comment:
This should go further down. The documentation for complex behavior is arguably more relevant for most users than the arXiv link.

Maybe use a `References:` section?

Reply:
Example: https://github.com/google/jax/blob/8278a72b885f65546487284aed4494c098cafd07/jax/_src/scipy/linalg.py#L1806-L1813

Reply:
@jakevdp Done.